javaagent/src/test/groovy/io/opentelemetry/javaagent/classloading/ShadowPackageRenamingTest.groovy

```diff
@@ -16,9 +16,7 @@ class ShadowPackageRenamingTest extends Specification {
   static final String[] AGENT_PACKAGE_PREFIXES = [
```

Comment:
I'm also wondering what this list is for - it doesn't seem to be updated (was missing jackson, has disruptor which we don't use)

Reply:
> I'm also wondering what this list is for

ya, me too, see the comment below 😄

```
    // TODO (trask) agentClasspath.getAllClasses() is empty
    //  so this part of the test doesn't verify what it thinks it is verifying
```

Reply:
@laurit any idea if there's anything worth salvaging from the `"agent jar contains no bootstrap classes"` test, or should we just delete it (and `AGENT_PACKAGE_PREFIXES`)